app/tests/unit/utils/time.test.ts

```diff
@@ -1,8 +1,9 @@
 import { calculateElapsedTime } from '@utils/time'
 
 describe('calculatElapsedTime()', () => {
+  const unixTime = 1689941745
```

Comment:
I think it would be great to add as a comment what date time does that unix time refer to.